MOSH_README.md

```diff
@@ -0,0 +1,64 @@
+# Using `mosh` with Fluent Terminal
+
+`mosh` can be used in Fluent Terminal in two ways:
```

Comment:
No space in FluentTerminal

Reply:
Changed in https://github.com/felixse/FluentTerminal/pull/572